adafruit_midi/midi_message.py

```diff
@@ -155,14 +175,17 @@ def _search_eom_status(cls, buf, eom_status, msgstartidx, msgendidxplusone, endi
         return (msgendidxplusone, good_termination, bad_termination)
 
     @classmethod
-    def _match_message_status(cls, buf, msgstartidx, msgendidxplusone, endidx):
+    def _match_message_status(
+        cls, buf: bytearray, msgstartidx: int, msgendidxplusone: int, endidx: int
+    ) -> Tuple[Optional[Any], bool, bool, bool, bool, int]:
```

Comment:
I think the first of the `bool`s in the return is for the `status` variable that gets returned, and a few lines down from here that is set as:
```
status = buf[msgstartidx]
```
So `status` should be an `int` I think. Making the full return type:
```
Tuple[Optional[Any], int, bool, bool, bool, int]:
```

I'm also not certain about the first type. Ideally there will be something more specific than Any for `cls` but I'm not sure how that works.